app/lib/actions/katello/content_view/incremental_updates.rb

```diff
@@ -67,7 +67,66 @@ def run
         end
 
         def humanized_name
-          _("Incremental Update")
+          total_count = total_counts(input)
+          _("Incremental Update of %{content_view_count} Content View Version(s) " %
+            {:content_view_count => total_count[:content_view_count]}) + content_output(total_count)
+        end
+
+        def total_counts(input)
+          total_count = {}
+          if input[:version_outputs]
+            content_view_count = input[:version_outputs].length
+            input[:version_outputs].map do |version_output|
+              added_units = version_output.try(:[], :output).try(:[], :added_units)
+              if added_units
+                total_count[:errata_count] = added_units[:erratum].try(:count)
+                total_count[:rpm_count] = added_units[:rpm].try(:count)
+                total_count[:puppet_module_count] = added_units[:puppet_module].try(:count)
+              end
+            end
+          end
+          total_count[:content_view_count] = content_view_count
+          total_count
+        end
+
+        def content_output(total_count)
+          content = content_output_collection(total_count)
+          if content.count >= 1
+            output = _("with")
```

Comment:
I'd probably refrain from using 'output' here, as it overrides a global method 'output' (as in task input/output)